config/initializers/catalog_admin.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class CatalogAdmin
```

Comment:
Where is this being used? I don't see it pop up anywhere.

Reply:
This isn't being used yet. I stole this from scholarsphere so we can wrap routes / namespaces in a constraint. I wanted to do something like `return unless CatalogAdmin` in the minitest authorizer, but couldn't get it to work.

Reply:
@whereismyjetpack You keep marking this as resolved, but I'm not seeing CatalogAdmin used anywhere. I think you want this in routes.rb to enable route-based restrictions, but that doesn't seem to be happening yet.